packages/nextjs/scaffold.config.ts

```diff
@@ -0,0 +1,22 @@
+import * as chains from "wagmi/chains";
+
+type ScaffoldConfig = {
+  targetNetwork: chains.Chain;
+  pollingInterval: number;
+  alchemyApiKey: string;
+};
+
+const scaffoldConfig: ScaffoldConfig = {
+  // The network where your DApp lives in
+  targetNetwork: chains.goerli,
+
+  // The interval at which your front-end polls the RPC servers for new data
+  // it has no effect on the local network
+  pollingInterval: 30000,
+
+  // This is ours Alchemy's default API key.
+  // You can get your own at https://dashboard.alchemyapi.io
+  alchemyApiKey: "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF",
```

Comment:
Update at [a5233c8](https://github.com/scaffold-eth/se-2/pull/246/commits/a5233c88f33ff058520981f79b666026f60c1cf8). 

Had to use `NEXT_PUBLIC` since it is being utilized at frontend.

NITPICK: This keys will be visible on the browser not directly but you can access them, Maybe in the future we can mention in README.md using `<details>` tag pointing to references of different strategies to hide them

Reply:
> NITPICK: This keys will be visible on the browser not directly but you can access them, Maybe in the future we can mention in README.md using <details> tag pointing to references of different strategies to hide them

I agree! But even if the key will be visible in the browser, I think it's a good practice to have these values in env vars instead of hardcoded.